packages/block-editor/src/components/block-tools/rich-text-tools.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton } from '@wordpress/components';
+import { formatBold, formatItalic } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { displayShortcut, isKeyboardEvent } from '@wordpress/keycodes';
+import { useSelect, useDispatch, useRegistry } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import groups from '../block-controls/groups';
+import { store as blockEditorStore } from '../../store';
+
+const formats = [
+	{
+		name: 'core/bold',
+		shortcutType: 'primary',
+		shortcut: 'b',
+		icon: formatBold,
+		title: __( 'Bold' ),
+	},
+	{
+		name: 'core/italic',
+		shortcutType: 'primary',
+		shortcut: 'i',
+		icon: formatItalic,
+		title: __( 'Italic' ),
+	},
```

Comment:
At some point I'd like to refactor rich text format registration to work without `edit` functions, much like how we'd envision a Block API (and data view fields) without `edit` functions.